src/clients/mod.rs

```diff
@@ -7,6 +7,8 @@ pub use oauth::OAuthClient;
 
 use crate::ClientResult;
 
+use std::fmt::Write as _;
```

Comment:
I am a little confused, what does this statement mean?

Reply:
It just imports the trait without getting it into the namespace. We just want to be able to use the methods in the trait, but not really `Write` itself, as it may collide or get in the way.